<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPacsConfigurationScreenBase</class>
 <widget class="QWidget" name="QPacsConfigurationScreenBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PACS Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../main/themes/starviewer/starviewer.qrc">
    <normaloff>:/images/icons/preferences-system-network.svg</normaloff>:/images/icons/preferences-system-network.svg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabConfiguration">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>PACS Servers</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableView" name="m_pacsTableView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="m_buttonAddPacs">
           <property name="toolTip">
            <string>Add a new PACS to the list</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../main/themes/starviewer/starviewer.qrc">
             <normaloff>:/images/icons/list-add.svg</normaloff>:/images/icons/list-add.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_buttonEditPacs">
           <property name="toolTip">
            <string>Edit parameters of the selected PACS</string>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
           <property name="icon">
            <iconset resource="../main/themes/starviewer/starviewer.qrc">
             <normaloff>:/images/icons/pencil.svg</normaloff>:/images/icons/pencil.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_buttonDeletePacs">
           <property name="toolTip">
            <string>Delete selected PACS</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../main/themes/starviewer/starviewer.qrc">
             <normaloff>:/images/icons/edit-delete.svg</normaloff>:/images/icons/edit-delete.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage_2">
      <attribute name="title">
       <string>Local Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Local DICOM Configuration</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel3">
           <property name="text">
            <string>Local Machine AE Title:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_textAETitleMachine">
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel5">
           <property name="text">
            <string>Max. Seconds for Timeout:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_textTimeout">
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel6">
           <property name="text">
            <string>Incoming Connections Port:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="m_textLocalPort">
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="textLabel7">
           <property name="text">
            <string>Max. Simultaneous Connections:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="m_textMaxConnections">
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2" rowspan="3">
          <widget class="QFrame" name="m_warningFrameIncomingConnectionsPortInUse">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">image: url(:/images/icons/emblem-warning.svg)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>This port is in use by another application.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Institution Contact Information</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Institution:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="m_textInstitutionName">
           <property name="maxLength">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Address:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="m_textInstitutionAddress">
           <property name="maxLength">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>City:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="m_textInstitutionTown">
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>ZIP Code:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="m_textInstitutionZipCode">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Country:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QLineEdit" name="m_textInstitutionCountry">
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="m_textInstitutionEmail">
           <property name="maxLength">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Phone Number:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="m_textInstitutionPhoneNumber">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main/themes/starviewer/starviewer.qrc"/>
 </resources>
 <connections/>
</ui>
